### ISSUE TITLE
Hindi experiment: - Var A - Split recommendations

### ISSUE THREAD

**Issue by JonBeeb** [bbc/simorgh#9869]
**Is your feature request related to a problem? Please describe.**
Overview

This will show promos split across 2 components, with 2 promos in each.

The first component will show promos for recs 1 and 2 and the second component will show promos for recs 3 and 4

These will be shown after the 5th and 9th paragraphs (if available)

The first component can be show without the second one if there is not enough paragraphs

Each component will have a different title 

Up to 4 promos will be shown in order based on the recommendations endpoint

The second component will only be show if there are 3 or more recommendations available

Optimizley view and click events will need to be added to the blocks

The view event will be sent if either block is on the screen (only 1 view event is sent).

The click event will be sent if promos from either block is clicked.

There will be a max of 2 components so can have a separate config for each

Check how the second component fit on the page if the page also includes the Inline Podcast Promo

Q: Can we send events this way for the split component? If not can we see if we can reuse something we have done to send the events (we can check with Jem what is acceptable)

 UX
Designs: https://zpl.io/aXyyrE8

A11y
This should be the same as what’s live for standard recs https://app.zeplin.io/project/6216b3ee60424c97dc083cc0/screen/6216b4b160424c97dc083d93

This component has a skip link anchor tag. We need to ensure that the skip link behaviour is correct for both components.

Test
Use the whitelist in Optimizley to show variation on Test

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Checklist**
- [ ] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Additional context**

**Comment by MuhammadAHussain** [comment]
Got a question regarding the view/click tracking, we are only doing optimizely tracking and not ATI tracking as well, correct?

**Comment by JonBeeb** [comment]
To keep this in line with the other variations, we should only fire 1 view event if either component is seen. 
I think to keep this simple, we only fire a view  event on the top component?   This would avoid any logic of working out if the 'other' component has fired an event on the page. We still fired clicks from both.

For ATI we keep it the same, fire 1 view event for the top component, but keep the rest (link view/clicks and component click) the same?  

Basically 'suppress the view event on the second component'

**Comment by amoore108** [comment]
Title to be used for the first block of recommendations:
```
  const title = pathOr(
    'You may also be interested in',
    ['recommendationTitle'],
    translations,
  );
```

Editorial to provide a translation for the second block of recommendations

**Comment by JonBeeb** [comment]
The (final) text will be
Recommended Stories and More Recommended Stories 
(waiting for translations)

**Comment by JonBeeb** [comment]
Recommended stories — ये भी पढ़ें
More recommended stories — कुछ और जानिए
